### DISCUSSION TITLE
Show more than 10 devices/browsers

### DISCUSSION THREAD

**Discussion by nicbou** [plausible/analytics#2613] [Anything Else]
I want to see how many users will be affected by code that only works in Safari 12+ (for example). This is not possible because I can't see more than 10 browser/device results at once. I would like to expand the device list like I can with other panels.

**Comment by metmarkosaric** [comment]
thanks for the feedback @nicbou, appreciate it! would be useful for sure!

**Comment by metmarkosaric** [comment]
There’s now the Details view in the Devices report which allows you to see the full list of browsers, browser versions, OS and OS versions.
